### PR TITLE
Babel6

### PR DESCRIPTION
**Pull request by jspears**
Guessing that moving to babel6 has a lot more logistics than just this, but just in case.  Here is a pull request.   Added a few tests and fixed the linting.   Also added a new prop for setting the expand collapse label.      Sorry for the massive PR, but this being Open Source and all you get what you pay for;)

Thanks for the component, really helpful.

### CODE REVIEW

**Comment by kenwheeler** [comment]
Can you squash those commits? Thanks so much for the PR!

**Comment by jspears** [comment]
Squashed.  Also changed the build phase to prepublish so a consumer does
not have to download babel to install the component successfully.
Thanks.
-js

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "presets": [
+    "react",
+    "es2015",
+    "stage-0"
+  ],
+  "env": {
+    // this plugin will be included only in development mode, e.g.
```
Comment (knowbody) [inline .babelrc]: can we remove those comments from here please?

**Comment by knowbody** [comment]
@jspears thank you for your work! this is great

**Comment by alexlande** [comment]
Going to close this one out as the babel 6 upgrade ended up landing in a separate PR.

**Pull request closed without merging**